skribenten-web/bff/.nais/bff-prod.yaml

```diff
@@ -0,0 +1,25 @@
+app-name: "skribenten-web"
+
+ingresses:
+  - "https://pensjon-skribenten.intern.nav.no"
+
+env:
+  - name: NODE_ENV
+    value: "production"
+  - name: EXPRESS_HOST
+    value: "::"
+  - name: EXPRESS_PORT
+    value: "8080"
+  - name: SKRIBENTEN_API_URL
+    value: "http://skribenten-backend"
+  - name: SKRIBENTEN_API_SCOPE
+    value: "api://prod-gcp.pensjonsbrev.skribenten-backend/.default"
+
+groups:
```

Comment:
Må ikke alle gruppene som @routsi bruker til tilgangsstyring med her?

Reply:
Hm, tror det ja. Da burde de jo inn i dev også om det skal funke der. Er det testa i dev?

Reply:
Jeg tror ikke de må det. Så la oss teste uten.

Reply:
Jeg tror gruppene vi skal ha her er de som skal gi tilgang til skribenten, ikke de som gir ekstra tilgang som kode6/7.

Reply:
Spørs vel hvor i løypa, brukes de gruppene til å validere claims i tokenet som kommer fra BFF, eller brukes gruppene  i videre OBO fra skribenten?

Reply:
De gruppene er kun med i skribenten-backend for at de skal bli inkludert i tokens Azure usteder til skribenten-backend. På den måten kan vi sjekke om brukeren har tilgang til å se kode6/7.